starboard/build/config/modular/BUILD.gn

```diff
@@ -147,7 +147,7 @@ config("modular") {
     ]
   }
 
-  if (is_clang_16) {
+  if (is_clang_16 || is_host_win) {
```

Comment:
Isn't this too broad? I can see this failing on other windows platforms which have clang < 15

Reply:
all windows modular host platforms will use the same cobalt toolchain with the same vs2022 compiler which is clang15 at the moment